Sources/TuistSupport/Utils/FileHandler.swift

```diff
@@ -140,6 +145,7 @@ public class FileHandler: FileHandling {
     }
 
     public func replace(_ to: AbsolutePath, with: AbsolutePath) throws {
+        log(to, with)
```

Comment:
Nitpick: What about calling the method print to stay consistent with the interface of the `Printer` utility?